app/main.py

```diff
@@ -1,5 +1,72 @@
+from typing import List, Any
+
+
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: List[int] = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords or [0, 0]
+
+    def go_forward(self, step: int = 1) -> None:
+        self.coords[1] += step
+
+    def go_back(self, step: int = 1) -> None:
+        self.coords[1] -= step
+
+    def go_right(self, step: int = 1) -> None:
+        self.coords[0] += step
+
+    def go_left(self, step: int = 1) -> None:
+        self.coords[0] -= step
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: List[int] = None
+    ) -> None:
+        super().__init__(name, weight, coords)
```

Comment:
```suggestion
        super().__init__(name, weight)
```

You add this argument in any case, so don't put it in super.